helm-charts/helm-api/templates/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       containers:
-      {{ if .Values.auditService.config.enabled }}
+      {{ if eq .Values.auditService.config.enabled "true" }}
```

Comment:
don't think `"true"` should be surrounded with quotes. It's causing a helm linting error that's affecting the PROW test